Explorer/Assets/DCL/Interaction/Systems/ProcessPointerEventsSystem.cs

```diff
@@ -167,6 +163,15 @@ private void ResetHighlightComponent(ref HighlightComponent highlightComponent)
                 // Try Append Hover Feedback
                 HoverFeedbackUtils.TryAppendHoverFeedback(sdkInputActionsMap, pointerEvent,
                     ref hoverFeedbackComponent, anyInputInfo.AnyButtonIsPressed);
+
+            }
+
+            if (!isAtDistance) return false; //This only works if elements have PBPointerEvents to filter distances, otherwise, we need to send the input event anyway (not done yet)
```

Comment:
is the fix "not done yet" still?

Reply:
this only fixes sending events to entities that have already a PointerEvents component but not to every entity that receives a raycast...